integrations/google_vertex/src/haystack_integrations/components/generators/google_vertex/gemini.py

```diff
@@ -1,14 +1,14 @@
 import logging
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
 
 import vertexai
```

Comment:
`import vertexai` should not be necessary. `from vertexai import init` or something like `from vertexai import init as vertexai_init` should be enough.